### ISSUE TITLE
State of PrBoom

### ISSUE THREAD

**Issue by fschulze** [coelckers/prboom-plus#41]
Hi!

I'm the original PrBoom guy and still host the SVN repository of it at https://svn.prboom.org/repos/

The last activity there was Sep 27th 2019 by entryway but since the end of 2017 it was almost inactive (besides some updates by @fabiangreffrath in 2018 via entryway). There seems to be quite some activity here though. Can anyone tell me who is currently working on it and doing releases? What is considered the canonical source nowadays?

I would like to update the stuff I have access to (sourceforge and prboom.org) to point to current stuff.

Maybe there is also interest in a prboom organization on github and using github pages for prboom.org?

Looking forward to a reply.

**Comment by coelckers** [comment]
Hi!

I am currently the maintainer of this fork, but as you might have discovered, it is suffering from getting recognition as being an official continuation of PrBoom+, mainly because the old Sourceforge repo still seems to be considered the canonical version.

However, with Entryway basically having ceased working on it, there's definitely some need to do something - as the author of another port this can inevitably only be a side project for me, and TBH, considering its stated goal as a classic oriented and demo compatible port, future additions should be weighed carefully to not dilute the ports main purpose, so I wouldn't expect this to become a busy project anyway.

An official PrBoom organization on Github would be an incredible boost to give continued work on the port an appearance of being the canonical continuation of the original PrBoom. Doing it on Github is definitely preferred over Sourceforce, collaboration here is just so much easier.

I'm definitely all for doing such a thing, but will wait for others to state their opinion as well. Sourceforge is a place that at some point in the past was a nice development hub but these days I don't think it's doing a project any favor anymore - Github would definitely be a better choice now.

Regarding future work, there are no plans to reinvent the port - but it really could use some careful extensions to support those features that have become a common base in most other ports - provided, of course, that nothing of this will affect its excellent demo playback capabilities.

**Comment by fabiangreffrath** [comment]
I am all for having this project community-maintained on Github within a dedicated organization (compare e.g. chocolate-doom) and have this recognized as the official continuation of both PrBoom and PrBoom+.

**Comment by Altazimuth** [comment]
@fschulze I like the idea of an organisation. Who would you think would be a good selection of people to be in charge of such an org?

**Comment by kraflab** [comment]
If this becomes the official continuation, it would be great if there was a speedrunner involved in the future to make sure something like the `2key sr50` issue doesn't creep up again... :sweat_smile: 

A direct contact to review features with competition implications could be handy. Someone who holds compatibility sacred :wink: 

4shockblast or zero-master would be good for such a role if they were willing.

**Comment by oleksiykamenyev** [comment]
Yeah, since PrBoom+ is a speedrunner-oriented port, having a speedrunner involved would be a good idea. I'd be willing to review features if needed

**Comment by runlow** [comment]
My 2 cents (as requesting opinion):

Please do make a github page, organisation (and maybe a world-editable wiki) - if/when you can, @coelckers

And perhaps if no longer interested in maintaining this repository - given that it's still maintained - maybe it would be a good idea to hand it over to @fabiangreffrath if he's happy enough? He seems to be doing a good job.

@fschulze perhaps for the time being you could link to this fork's repo as it is now - on [prboom.org](http://www.prboom.org/) as well as [sourceforge](https://sourceforge.net/p/prboom-plus/)? Latest news on [prboom.org](http://www.prboom.org/)  are from 13 years ago which could lead a visitor (as it did for me in the past) to think the sourceport is no longer maintained. I understand that ideally you'd like to forward to a page/organisation but perhaps in the meanwhile people could just land here?

**Comment by fabiangreffrath** [comment]
> And perhaps if no longer interested in maintaining this repository - given that it's still maintained - maybe it would be a good idea to hand it over to @fabiangreffrath if he's happy enough? He seems to be doing a good job.

Thank you, but this shouldn't be *my* repository. I am all in favor of creating a Github organization and passing the repository over. If no new organization is going to be created, maybe we could use a more or less related existing one, e.g. https://github.com/chocolatiers. What do you say, @chungy?

> @fschulze perhaps for the time being you could link to this fork's repo as it is now - on [prboom.org](http://www.prboom.org/) as well as [sourceforge](https://sourceforge.net/p/prboom-plus/)? Latest news on [prboom.org](http://www.prboom.org/) are from 13 years ago which could lead a visitor (as it did for me in the past) to think the sourceport is no longer maintained. I understand that ideally you'd like to forward to a page/organisation but perhaps in the meanwhile people could just land here?

Please don't forget to also lead people to this port's brother fork at https://github.com/kraflab/dsda-doom maintained by @kraflab. @andrey-budko already linked both projects on the former PrBoom+ project page.

**Comment by fschulze** [comment]
I have already created https://github.com/prboom I will look into moving stuff there and adding people as time permits. Who would want to be added and why?

I would love to do a proper cleaned up git conversion of the PrBoom SVN repository, but this might just be my perfectionism getting in the way. It would have been much more interesting if I would have got the Boom CVS repository history. Unfortunately the backup of it was lost several years ago due to a burst pipe. I contacted some people about it in 2013.

I will also look into making prboom.org hosted from a GitHub repository.

**Comment by fschulze** [comment]
Oh, I just read the emails from 2013 and noticed that there could be one last lead. Lee Killough might still have a backup of Boom CVS, if anyone has a way to contact him, maybe it can be dug up.

**Comment by runlow** [comment]
@fschulze don't know if this helps but [Lee seems to have made a doomworld account](https://www.doomworld.com/profile/22843-lee-killough/) relatively recently. If you have an account there perhaps you could try to DM him about a backup of Boom CVS?
